dictionaries/CallMap_72_delta.php

```diff
@@ -34,6 +34,7 @@
     'ldap_exop_whoami' => ['string|false', 'ldap'=>'resource'],
     'ldap_parse_exop' => ['bool', 'ldap'=>'resource', 'result'=>'resource', '&w_response_data='=>'string', '&w_response_oid='=>'string'],
     'mb_chr' => ['string|false', 'codepoint'=>'int', 'encoding='=>'string'],
+    'mb_convert_encoding\'1' => ['array', 'string'=>'array', 'to_encoding'=>'string', 'from_encoding='=>'mixed'],
```

Comment:
Oh, nice catch! I didn't know arrays were introduced in 7.2

Reply:
At least the manual says so :-)